src/applications/static-pages/sass/modules/_m-checklist-layout.scss

```diff
@@ -0,0 +1,7 @@
+.va-l-detail-page--checklist {
+  @media print {
+    #resources-search-bar-wrapper {
```

Comment:
@ncksllvn can we use a class instead of an ID so the specificity isn't so aggressive?

Reply:
I think this is a good use case for an ID - a single instance of a unique element on the screen

Reply:
I'd also probably choose a class here, it being an `id` makes me think there's some javascript associated with that element (that's probably doing `document.getElementById`) as an outside dev 🙂 but up to you!

Reply:
I know we don't use Tailwind CSS, but this conversation has sent me down a rabbit hole to see how other CSS utility libraries handle `print`. 

TIL https://tailwindcss.com/docs/breakpoints#styling-for-print

Reply:
That's a cool utility but it seems like it would have a lot of bulk in the outputted CSS and limited use

Reply:
@kelsonic it's the same concept though - an outside script grabbing a reference to a unique component to manipulate it. If we were going to apply this style to more than one component on the site then I would be convinced that a class is the right thing to do though.

Reply:
Makes sense! 🙂